fabtests/man/man7/fabtests.7

```diff
@@ -307,6 +307,12 @@ Tests memory registration.
 Allocates and closes fabric resources to check for proper cleanup.
 .RS
 .RE
+.TP
+.b \f[I]fi_multinode\f[]
+Runs a full mesh, all to one, one to all, and ring patterns on multiple 
+procceses over sockets, rma, atomics, and tagged messages.
```

Comment:
Let's call this documentation lite lite...  Can we get a more complete description of what this test does and how to use it?

Reply:
I did this in the md file. Do you also want me to add the same information here?

Reply:
Here we can have a simple explanation, such as:
```
Performs a series of data transfers between 2 or more nodes.  Used for validating at scale.
```

Reply:
oh, I guess I'll just take this out then